rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -53,6 +53,7 @@
 import org.mockito.ArgumentCaptor;
 import org.mockito.Mockito;
 
+import javax.annotation.Nullable;
```

Comment:
?

Reply:
Move Nullable parameter to the end of list of parameters, but left it just for clarity.